### ISSUE TITLE
Video output on iOS

### ISSUE THREAD

**Issue by AnByronV** [flutterwtf/VR-Player#44]
Hi there,
I was trying to reproduce the video, but, the output image is not woking. The problem should be in my video because the video example it's working.
This is the video example that works: https://cdn.bitmovin.com/content/assets/playhouse-vr/m3u8s/105560.m3u8
And this is my video: https://vrestudio.com/storage/AppAlmeria/Video360/EmeritaLudica/Emerita_Ludica.m3u8

This is the output warning when I run on Xcode:

```
"[MonoSphericalVideoScene] failed to render video with error: The pixel buffer is not compatible with Metal due to an unsupported buffer size, pixel format, or attribute."
```

Note: All works fine on Android.

**Comment by marcoberetta96** [comment]
any update on this?

**Comment by AnByronV** [comment]
No, but my problem was solved changing the aspect ratio of video to 16:9

**Comment by marcoberetta96** [comment]
you are using VRMode?

For me everything is fine on Android, but sx and dx videos are not aligned on iOS :(

**Comment by marcoberetta96** [comment]
@AnByronV so you do not have this issue? https://github.com/flutterwtf/VR-Player/issues/52

**Comment by AnByronV** [comment]
@marcoberetta96 Hi,yes I have the same problem on IOS
![image](https://github.com/user-attachments/assets/057255d8-9ef1-4d5c-a3e2-40255f8c2d7e)

**Comment by marcoberetta96** [comment]
@AnByronV no, I trying both to understand why iOS has that behaviour and how to fix it
